tools/c7n_gcp/c7n_gcp/resources/pubsub.py

```diff
@@ -34,3 +34,31 @@ class resource_type(TypeInfo):
         def get(client, resource_info):
             return client.execute_command(
                 'get', {'topic': resource_info['topic_id']})
+
+
+@resources.register('pubsub-subscription')
+class PubSubSubscription(QueryResourceManager):
```

Comment:
uh.. this seems questionable.. whats the use case for a pubsub-subscription resource...